.github/workflows/azure-static-webapp.yml

```diff
@@ -53,6 +53,7 @@ jobs:
 
       - name: Publish to Azure Static WebApps
         id: builddeploy_uno
+        continue-on-error: true
```

Comment:
Why is this needed now? The PR checks should pass anyways, I removed the requirement for that step.

Reply:
Ah, I see, I didn't notice the reason the merging was blocked was approve missing.